### DISCUSSION TITLE
Is there a yt-dlp --no-hls-use-mpegts equivalent in streamlink?

### DISCUSSION THREAD

**Discussion by chorong761** [streamlink/streamlink#5232] [Q&A] [answered]
### Description

Is there a yt-dlp --no-hls-use-mpegts equivalent in streamlink?

I want to capture the file as .mp4 but I don't want it to mux it with ffmpeg at the end.

**Comment by bastimeyer** [comment, marked as answer]
Streamlink doesn't do any remuxing of its output unless muxing is required, for example when a different or an additional audio stream needs to be used, or when a subtitle stream gets included, or in case of DASH or other stream implementations where video and audio streams are separate. When muxing is required, users can choose a different container format via `--ffmpeg-fout`, but this won't alter the container format of unmuxed streams, like regular segmented streams like HLS or progressive HTTP streams:  
https://streamlink.github.io/cli.html#cmdoption-ffmpeg-fout
